server/middleware/auditMiddleware.ts

```diff
@@ -23,6 +23,10 @@ const pageViewEventMap: Record<string, Page> = {
   '/plan/:prisonNumber/goals/:goalReference/update': Page.UPDATE_GOALS,
   '/plan/:prisonNumber/goals/:goalReference/update/review': Page.UPDATE_GOALS_REVIEW,
 
+  // Archive goals
```

Comment:
Ah, I'm glad you found this. I forgot to tell you about this (its quite a recent addition that Lee added)
Without this change I suspect you would have been getting 404's when trying to request your new pages?

Reply:
yeah spent a bit too long figuring it out TBH. There was a hint in the logs but though I had the mappings wrong for a while!